### PR TITLE
- General clarifications and some corrections

### PR DESCRIPTION
**Pull request by rjschwei**

### CODE REVIEW

**Comment by rjschwei** [comment]
@cwickert ready for your review

File: xml/cha_images.xml [hunk 1]
```diff
@@ -83,23 +84,26 @@
...
    </para>
   </sect2>
 
   <sect2 xml:id="sec-image-naming">
    <title>Naming scheme</title>
    <para> Names for &suse;'s public cloud images consist of multiple parts
-    that contain information about the product, it's version, intended cloud
-    framework, and more. The general naming scheme for &suse;'s public
-    cloud images is as follows: </para>
+   that contain information about the product, it's version, a time stamp
```
Comment (cwickert) [inline xml/cha_images.xml]:
```suggestion
   that contain information about the product, its version, a time stamp
```
(This was my typo and this comment is just a reminder to fix it after merging)

File: xml/cha_images.xml [hunk 2]
```diff
@@ -136,28 +145,35 @@
...
-     <term><literal>ARCHITECTURE</literal> (&awsa;-only)</term>
+     <term><literal>ARCHITECTURE</literal></term>
      <listitem>
       <para>
-       Either <literal>i386</literal>,
-       <literal>x86_64</literal>, or <literal>arm64</literal>.
+        Either <literal>x86_64</literal> or <literal>arm64</literal>.
+        &suse; no longer supports or publishes 32 bit <literal>x86</literal>
+        images. Images with the <literal>i386</literal> identifier are visible
+        in &pint; data.
```
Comment (cwickert) [inline xml/cha_images.xml:176]: I think we should align this text with the others.

Is the last sentence really relevant any longer? All of the i386 images are deleted and not available to create new instances. They were ancient (the last ones from 2015) and deleted two years ago. I suggest we remove that sentence. OK?

File: xml/cha_images.xml [hunk 3]
```diff
@@ -449,17 +469,19 @@
...
     </varlistentry>
     <varlistentry>
      <term>Type</term>
      <listitem>
       <para>
-       One of <literal>regionserver</literal> or <literal>smt</literal>. Every
-       Region has one <literal>regionserver</literal> and several
-       <literal>smt</literal> servers.
+        One of <literal>regionserver</literal> or <literal>smt</literal>.
+        In every framework where &suse; operates an update infrastrutcure the
```
Comment (cwickert) [inline xml/cha_images.xml]:
```suggestion
        In every framework where &suse; operates an update infrastructure, the
```

File: xml/cha_images.xml [hunk 4]
```diff
@@ -477,18 +499,18 @@
...
     </para>
    </listitem>
    <listitem>
     <para>
-     If a security vulnerability occurs during the three months that needs to be
-     addressed, affected images in <literal>active</literal> and
-     <literal>inactive</literal> states are updated as soon as possible once the
-     fix for the affected code is available. The three month timer restarts with
-     this forced replacement.
+      If a critical security vulnerability occurs images in
```
Comment (cwickert) [inline xml/cha_images.xml]:
```suggestion
      If a critical security vulnerability occurs, images in
```

**Review by cwickert — APPROVED** [review]
Just some typos and the indention, but I'll take care of that after merging and mark them as resolved here. We then only need to think about the remaining 1-2 questions.

File: xml/cha_images.xml (hunk 2 above)
Comment (rjschwei) [reply xml/cha_images.xml:176]: I generally have no problem removing the `i386` mention here, but we should reference it somewhere as those images are still in the pint data. A user of pint listing everything will find `i386`. It should be obvious what it means but we probably should not make that assumption.

File: xml/cha_images.xml [hunk 5]
```diff
@@ -117,10 +121,15 @@
...
-       <!-- FIXME cwickert 20210830: is rightscale still a thing?! -->
-       e.g. <literal>sap</literal> or
-       <literal>rightscale</literal>. If it is the default image of a product,
-       this part will be omitted.
+       e.g. <literal>chost</literal> or
+       <literal>byos</literal>. If it is the default image of a product,
+       this part will be omitted. Multiple <literal>FLAVOR</literal> attributes
+       may be combined in an image name. For example
+       <literal>sles-15-sp3-chost-byos</literal>
+       Is an image build based on &sles; 15 SP3 build as a container host
```
Comment (cwickert) [inline xml/cha_images.xml]:
```suggestion
       is an image build based on &sles; 15 SP3 build as a container host
```

File: xml/cha_images.xml [hunk 6]
```diff
@@ -273,9 +289,12 @@
...
   <para>
-   The &pint; (&pinta;) provides information about the images and servers &suse;
-   publishes in supported public cloud frameworks. &pinta; is available at
-   <link xlink:href="&pinturl;"/>.
+   The &pint; (&pinta;) provides information about the images &suse;
+   publishes and servers that are part of the &suse; operated update
+   infrastructure. &pinta; is available at
+   <link xlink:href="&pinturl;"/> and provided as an API and commandline
+   tool with the <package>python3-susepubliccloudinfo</package> package
+   from the <repository>Public Cloud Module</repository> repository.
```
Comment (cwickert) [inline xml/cha_images.xml]: This does not validate, there is no `repository` tag in DocBook. Use `systemitem` or simply `literal` instead.
```suggestion
   from the <systemitem>Public Cloud Module</systemitem> repository.
```

**Review by cwickert — COMMENTED** [review]
Two mire fixes I'll approve.

**Comment by cwickert** [comment]
I merged this manually because I wanted to fix some more things before. Ignore the conflict here, all your changes are in `main` already.

**Pull request closed without merging**